src/current/v23.2/stored-procedures.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Stored Procedures
+summary: A stored procedure consists of PL/pgSQL or SQL statements that can be issued with a single call.
+toc: true
+key: sql-expressions.html
+docs_area: reference.sql
+---
+
+{% include_cached new-in.html version="v23.2" %} A *stored procedure* is a database object consisting of [PL/pgSQL]({% link {{ page.version.version }}/plpgsql.md %}) or [SQL]({% link {{ page.version.version }}/sql-statements.md %}) statements that can be issued with a single [`CALL`]({% link {{ page.version.version }}/call.md %}) statement. This allows complex logic to be executed repeatedly within the database, which can improve performance and mitigate security risks.
+
+{% include {{ page.version.version }}/sql/udfs-vs-stored-procs.md %}
+
+## Structure
+
+A stored procedure consists of a name, optional parameters, language, and procedure body.
+
+~~~ sql
+CREATE PROCEDURE procedure_name(parameters)
+  LANGUAGE procedure_language
+  AS procedure_body
+~~~
+
+- Each parameter can be a supported [SQL data type]({% link {{ page.version.version }}/data-types.md %}), [user-defined type]({% link {{ page.version.version }}/create-type.md %}), or the PL/pgSQL `REFCURSOR` type, when [declaring PL/pgSQL cursor variables]({% link {{ page.version.version }}/plpgsql.md %}#declare-cursor-variables).
+- `LANGUAGE` specifies the language of the function body. CockroachDB supports the languages [`SQL`]({% link {{ page.version.version }}/sql-statements.md %}) and [`PLpgSQL`]({% link {{ page.version.version }}/plpgsql.md %}).
```

Comment:
```suggestion
- `LANGUAGE` specifies the language of the function body. CockroachDB supports the languages [`SQL`]({% link {{ page.version.version }}/sql-statements.md %}) and [`PL/pgSQL`]({% link {{ page.version.version }}/plpgsql.md %}).
```

Reply:
This is specifically describing the syntax, so the forward slash is intentionally left out.